eureka_ml_insights/data_utils/nphard_tsp_utils.py

```diff
@@ -0,0 +1,35 @@
+import re
+from dataclasses import dataclass
+import ast
+import pandas as pd
+from eureka_ml_insights.data_utils import DFTransformBase
+import xml.etree.ElementTree as ET
+
+@dataclass
+class NPHARDTSPExtractAnswer(DFTransformBase):
+    model_output_column: str
```

Comment:
It would be great if you could leave some comments on what this code does, maybe a single line docstring for every method?